src/main/webapp/interval-form.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="">
```

Comment:
The attribute on this line should probably have a non-empty value such as `lang="en"`  This issue is found on all three of the *.html files in this PR.